CHANGES.md

```diff
@@ -1,3 +1,26 @@
+## edge-19.12.3
+
+* CLI
+  * Increased the comprehensiveness of `linkerd check --pre`
+  * Added TLS certificate validation to `check` and `upgrade` commands
+* Controller
+  * Increased minimum kubernetes version to 1.13.0
+  * Added support for pod ip and service cluster ip lookups in the destination 
+    service
+  * Added recommended kubernetes labels to control-plane
+  * Added preStop hook into the proxy sidecar container to stop it last (a huge 
+    commit from @KIVagant, thanks!)
```

Comment:
That's opt-in, so I would be more precise and say something like "Added the `--wait-before-exit-seconds` flag to `linkerd inject` for the proxy sidecar to delay the start of its shutdown process". Not sure if we'd like to also add more context on the cases where adding that delay makes sense.